tests/test_CFDataset.py

```diff
@@ -282,6 +282,30 @@ def test_filepath(cwd, raw_dataset, converter, expected):
     ('gridded_obs', 'unique_id',
      'pr_day_SYMAP_BC_v1_historical_19500101-19500104'),
 
+     ('gridded_mClimSD_obs', 'first_MiB_md5sum', '7eb975dfd17845621123400dbb6d0e5b'),
+     ('gridded_mClimSD_obs', 'md5', '7eb975dfd17845621123400dbb6d0e5b'),
+     ('gridded_mClimSD_obs', 'climatology_bounds_var_name', 'climatology_bnds'),
+     ('gridded_mClimSD_obs', 'sampling_geometry', 'gridded'),
+     ('gridded_mClimSD_obs', 'is_multi_year', True),
+     ('gridded_mClimSD_obs', 'is_multi_year_mean', True),
+     ('gridded_mClimSD_obs', 'time_range', (13163.0, 13253.0)),
+     ('gridded_mClimSD_obs', 'time_step_size', time_to_seconds(31, 'days')),
+     ('gridded_mClimSD_obs', 'time_resolution', 'seasonal'),
+     ('gridded_mClimSD_obs', 'is_unprocessed_gcm_output', False),
+     ('gridded_mClimSD_obs', 'is_downscaled_output', False),
+     ('gridded_mClimSD_obs', 'is_hydromodel_output', False),
+     ('gridded_mClimSD_obs', 'is_hydromodel_dgcm_output', False),
+     # ('gridded_mClimSD_obs', 'is_hydromodel_iobs_output', False), # TODO
+     ('gridded_mClimSD_obs', 'is_streamflow_model_output', False),
+     ('gridded_mClimSD_obs', 'is_streamflow_model_dgcm_output', False),
+     ('gridded_mClimSD_obs', 'is_streamflow_model_iobs_output', False),
+     ('gridded_mClimSD_obs', 'is_climdex_output', False),
+     ('gridded_mClimSD_obs', 'is_climdex_ds_gcm_output', False),
```

Comment:
Add test for `is_gridded_obs` here and in all other tests for example files. A bit tedious, but we do need to test that it works correctly.